Answers.md

```diff
@@ -0,0 +1,25 @@
+1. If you saw this HTML: ```<div class="box box1 box2 box3"></div>``` which class has the most specificity weight?
+
+The class with the most specificity weight is box box1 box2 box3, the specificity increases when there are more nested selectors.
```

Comment:
box3 would have the highest, the sibling furthest to the right holds the most weight